sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/OAuthCopySourceTests.groovy

```diff
@@ -0,0 +1,165 @@
+package com.azure.storage.blob
+
+import com.azure.core.experimental.http.HttpAuthorization
+import com.azure.core.util.BinaryData
+import com.azure.core.util.Context
+import com.azure.storage.blob.models.BlobStorageException
+import com.azure.storage.blob.models.PageRange
+import com.azure.storage.blob.options.AppendBlobAppendBlockFromUrlOptions
+import com.azure.storage.blob.options.BlobUploadFromUrlOptions
+import com.azure.storage.blob.options.BlockBlobStageBlockFromUrlOptions
+import com.azure.storage.blob.options.PageBlobUploadPagesFromUrlOptions
+import com.azure.storage.blob.specialized.AppendBlobClient
+import com.azure.storage.blob.specialized.BlockBlobClient
+import com.azure.storage.blob.specialized.PageBlobClient
+import com.azure.storage.common.test.shared.extensions.RequiredServiceVersion
+
+@RequiredServiceVersion(clazz = BlobServiceVersion.class, min = "2020-10-02")
```

Comment:
Please check the format for `min` , see https://github.com/Azure/azure-sdk-for-java/blob/b507f4a4dc70d7439635f433520473735a8bf6cd/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy#L234 .